File: AutoserviceOnlineClient/src/app/helpers/classes/models/car.ts
export class Car {
  private _id: number;
  private _manufacturer: string;
  private _model: string;
  private _vin: string;
  private _manufactureYear: number;
  private _customerId: number;


  constructor(id: number, manufacturer: string, model: string, vin: string, manufactureYear: number, customerId: number) {
    this.id = id;
    this.manufacturer = manufacturer;
    this.model = model;
    this.vin = vin;
    this.manufactureYear = manufactureYear;
    this.customerId = customerId;
  }

  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get manufacturer(): string {
    return this._manufacturer;
  }

  set manufacturer(value: string) {
    this._manufacturer = value;
  }

  get model(): string {
    return this._model;
  }

  set model(value: string) {
    this._model = value;
  }

  get vin(): string {
    return this._vin;
  }

  set vin(value: string) {
    this._vin = value;
  }

  get manufactureYear(): number {
    return this._manufactureYear;
  }

  set manufactureYear(value: number) {
    this._manufactureYear = value;
  }

  get customerId(): number {
    return this._customerId;
  }

  set customerId(value: number) {
    this._customerId = value;
  }
}

File: DataAccess/Model/Car.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;

namespace DataAccess.Model
{
    [Table("autoservicedb2.car")]
    public partial class Car
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public Car()
        {
            Order = new HashSet<Order>();
        }

        public int Id { get; set; }

        [Required]
        [StringLength(20)]
        public string Manufacturer { get; set; }

        [Required]
        [StringLength(20)]
        public string Model { get; set; }

        [Required]
        [StringLength(17)]
        public string Vin { get; set; }

        public int ManufactureYear { get; set; }

        public int CustomerId { get; set; }

        [JsonIgnore]
        public virtual Customer Customer { get; set; }

        [JsonIgnore]
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<Order> Order { get; set; }
    }
}
